src/applications/check-in/day-of/pages/TravelReview.jsx

```diff
@@ -89,25 +98,33 @@ const TravelQuestion = props => {
           uswds
         >
           <div slot="description">
-            <p>{t('by-submitting-claim')}</p>
-            <va-additional-info
-              uswds
-              trigger={t('beneficiary-travel-agreement')}
-              class="vads-u-margin-bottom--3"
-            >
-              <span className="vads-u-font-weight--bold">
-                {t('please-review')}
-              </span>
-              <ul>
-                <Trans
-                  i18nKey="certify-statements"
-                  components={[<li key="list-item" />]}
-                />
-              </ul>
-            </va-additional-info>
+            <p>
+              <Trans
+                i18nKey="by-submitting-this-claim"
+                components={[
+                  <a
+                    data-testid="travel-agreement-link"
+                    key="link"
+                    aria-label={t('beneficiary-travel-agreement')}
+                    href="travel-agreement"
+                    onClick={e => agreementLink(e)}
+                  />,
+                ]}
+              />
+            </p>
           </div>
         </VaCheckbox>
       </div>
+      <div className="vads-u-margin-top--2p5">
+        <a
+          className="vads-u-margin-left--auto"
+          href="travel-mileage"
+          onClick={() => startOverAction()}
```

Comment:
```suggestion
          onClick={e => startOverAction(e)}
```
This isn't passing the event through, so it is still navigating to the href of `travel-mileage`